.github/workflows/release-marketing-app.yml

```diff
@@ -20,6 +20,9 @@ env:
 
 # There is a single job in this workflow. It's configured to run on the latest available version of Ubuntu.
 jobs:
+  # Job: build-and-push-image
+  # This job builds a docker image for the frontend repository and pushes the docker image to Github Packages.
```

Comment:
nit: are we using inconsistent terminology? What's "Marketing App" compared to "Frontend Repository"?